### PR TITLE
Brand styles room component

### PR DESCRIPTION
**Pull request by Estaffieri**
## Description
This PR builds out the room component, brings fetches the backdrop images from the API, fetches memories from the API, displays the backdrop and the memory image in a roughed out modal. This PR also explored the possibly of capturing click coordinates to specify where a memory should be placed within a room. These coordinates will be stored on the backend and pulled in for each room.

## Related Issue
#8 
#11 
#5 
#6 

## Motivation and Context
Each room is essentially a map for the memories in the mind palace.  Rooms hold memories that can be recalled when a specific object is clicked on! These do not have to be directly related to the object they are connected with. We needed to provide clickable areas that would bring up a modal with our memory images and any text for that memory.

## How Has This Been Tested?
This has been tested in an ios Simulator.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/Room/Room.js [hunk 1]
```diff
@@ -1,41 +1,77 @@
...
+
+  useEffect(() => {
+    getSelectedRoom();
+    getRoomMemories();
+  }, []);
+
+  const getSelectedRoom = async () => {
+    await getRoom(selectedRoom)
+      .then((data) => setRoom(data.data))
+      .then(() => {
```
Comment (brycemara) [inline src/Room/Room.js:21]: Remember later on to delete this - not doing anything.

**Review by brycemara — COMMENTED** [review]
Good progress. Next step is figuring out the thought bubble svg and rendering the modal.

File: assets/brandColors.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//==== Brand Colors 2/26/2021 ==== //
+// ==== Primary Brand Colors ==== //
```
Comment (cameronRomo) [inline assets/brandColors.js:2]: Love keeping the continuity of color this way!

File: src/Room/Room.js [hunk 3]
```diff
@@ -1,41 +1,77 @@
...
+      .then((data) => console.log(memories))
+      .catch((error) => console.error(error));
   };
 
   return (
-    <View style={styles.container}>
-      <Button
+    <TouchableOpacity
+      onPress={(event) =>
+        console.log(event.nativeEvent.locationX, event.nativeEvent.locationY)
```
Comment (cameronRomo) [inline src/Room/Room.js:37]: This is going to bring so much to the application. Very excited about this!

File: src/Room/Room.js [hunk 4]
```diff
@@ -1,41 +1,77 @@
...
     alignItems: "center",
     justifyContent: "center",
   },
+  button: {
+    zIndex: 2,
+    position: "absolute",
+    borderWidth: 4,
+    borderColor: "red",
+    bottom: 200,
+    left: 60,
```
Comment (Codo-Baggins) [inline src/Room/Room.js:73]: These two values are what we are making dynamic to change with the x and y coordinates that are coming in with each memory.

File: src/Memory/Memory.js [hunk 5]
```diff
@@ -1,29 +1,37 @@
...
       <Text>MEMORY</Text>
-      {/* <Image source={require(`${image}`)}></Image>
-      <Text> {description} </Text>
-      <Text>{aromas}</Text> */}
+      {/* <Modal source={{ uri: `${memory.image}` }} style={styles.memoryImage}> */}
+        <View>
+          <Text> {memory.description} </Text>
+          <Text>{memory.aromas}</Text>
+        </View>
+      {/* </Modal> */}
```
Comment (Codo-Baggins) [inline src/Memory/Memory.js:14]: We should make sure to change the style of the modal to take up a larger portion of the page.  We want users to be able to see the material very clearly.

**Review by Codo-Baggins — APPROVED** [review]
Nice job here Estelle!

**Pull request merged**